resolwe_bio/processes/workflows/cut_and_run.yml

```diff
@@ -0,0 +1,624 @@
+          type: basic:string
+          required: false
+          choices:
+            - label: BAM
+              value: "BAM"
+            - label: BAMPE
+              value: "BAMPE"
+          default: "BAMPE"
+          description: |
+            This specifies the format of input files. For paired-end data the format
+            dicates how MACS2 will treat mates. If the selected format is BAM, MACS2
+            will only keep the left mate  (5' end) tag. However, when format BAMPE is
+            selected, MACS2 will use actual insert sizes of pairs of reads to build
+            fragment pileup, instead of building bimodal distribution plus and minus
+            strand reads to predict fragment size.
+        - name: pvalue
+          label: P-value cutoff
+          type: basic:decimal
+          required: false
+          description: |
+            The p-value cutoff.
+          default: 0.001
+        - name: duplicates
+          label: Number of duplicates
+          type: basic:string
+          default: "all"
+          choices:
+            - label: "1"
+              value: "1"
+            - label: "auto"
+              value: "auto"
+            - label: "all"
+              value: "all"
+          description: |
+            It controls the MACS behavior towards duplicate tags at the exact same
+            location -- the same coordination and the same strand. The 'auto' option
+            makes MACS calculate the maximum tags at the exact same location based on
+            binomal distribution using 1e-5 as pvalue cutoff and the 'all' option
+            keeps all the tags. If an integer is given, at most this number of tags
+            will be kept at the same location. The default is to keep one tag at the
+            same location.
+        - name: bedgraph
+          label: Save fragment pileup and control lambda
+          type: basic:boolean
+          default: true
+          description: |
+            If this flag is on, MACS will store the fragment pileup, control lambda,
+            -log10(pvalue) and -log10(qvalue) scores in bedGraph files. The bedGraph
+            files will be stored in current directory named NAME+'_treat_pileup.bdg'
+            for treatment data, NAME+'_control_lambda.bdg' for local lambda values
+            from control, NAME+'_treat_pvalue.bdg' for Poisson pvalue scores (in
+            -log10(pvalue) form), and NAME+'_treat_qvalue.bdg' for q-value scores
+            from Benjamini-Hochberg-Yekutieli procedure.
+
+    - name: options_sieve
+      label: Alignment sieve options
+      group:
+        - name: min_frag_length
+          label: Minimum fragment length
+          type: basic:integer
+          description: |
+            The minimum fragment length needed for read/pair inclusion. This option
+            is primarily useful in ATACseq experiments, for filtering mono- or
+            di-nucleosome fragments. Default is 0.
+          required: false
```

Comment:
Could this be set to `default: 0` instead of `required: false`?

Reply:
All inputs have been set to their tool defaults when applicable.